### ISSUE TITLE
Huge CPU load

### ISSUE THREAD

**Issue by mikeywaites** [google/cadvisor#1897]
We have been setting up cadvisor/prometheus stack inside of an AWS ECS cluster and so far prometheus is reporting huge CPU load from the cadvisor container running inside of the cluster.

On avg its ~38% CPU load.  It was as high as 50% previously though it dropped after increasing the `house-keeping-interval` to 5s.

I've also tried disable the following metrics `--disable-metrics=disk  --disable_metrics=tcp --disable_metrics=udp` but that had no effect on the load.

Using docker stats on the instance where the cadvisor task is running i see the CPU usage of the container jump around like crazy.  it reports anywhere between 16% and 90%.

cadvisor version: v0.29.0
docker version: Docker version 17.06.2-ce, build cec0b72
Linux ip-172-45-72-151 4.4.0-1047-aws #56-Ubuntu SMP Sat Jan 6 19:39:06 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

Happy to provide any other information you need.

**Comment by dashpole** [comment]
Just to make sure, you passed `--disable-metrics='disk,tcp,udp'`.  I would recommend increasing housekeeping even higher to 10s as well.

**Comment by mikeywaites** [comment]
hey thanks for such a quick reply.

I did actually have the `--disable-metrics` flag configured wrong in the entry point which I've now resolved.

Here's what the output from docker inspect shows from entrypoint

```
"Entrypoint": [
                "/usr/bin/cadvisor",
                "--logtostderr",
                "--housekeeping_interval",
                "10s",
                "--disable_metrics",
                "disk,tcp,udp"
            ],
```

So it's come down ever so slightly.  It's not sitting around 27% cpu according to Prometheus.

Is there anything else I can try or provide you with ?
